jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/frames/PriorityFrame.java

```diff
@@ -41,6 +41,11 @@ public PriorityFrame(int streamId, int parentStreamId, int weight, boolean exclu
         this.exclusive = exclusive;
     }
 
+    public PriorityFrame(int streamId, PriorityFrame other)
+    {
+        this(streamId, other.getParentStreamId(), other.getWeight(), other.isExclusive());
+    }
+
```

Comment:
Remove if only used by the implementation.

Reply:
ok